### PR TITLE
Npm update

### PR DESCRIPTION
**Pull request by frankyhollywood**
https://thehyve.atlassian.net/browse/FNS-121
https://thehyve.atlassian.net/browse/FNS-120

- Postgres 13 -> 15
- Node 16 -> 18
- Alpine 3.9 -> 3.18

### CODE REVIEW

File: projects/mercury/jest-setup.js
Comment (ewelinajozwik) [inline projects/mercury/jest-setup.js:1]: Isn't this file needed only for your local setup?

File: local-development/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   fairspace-keycloak-postgres:
-    image: postgres:13-alpine
```
Comment (ewelinagr) [inline local-development/docker-compose.yml:5]: Could you also change it in Keycloak repo for consistency?

**Review by ewelinajozwik — COMMENTED** [review]
I tested with minikube, all works well, although we'll need to drop the database for deployment. 
Only one comment above.

File: projects/mercury/jest-setup.js
Comment (frankyhollywood) [reply projects/mercury/jest-setup.js:1]: No it isn't, polyfills were removed from Node 18 so you need to provide global TextEncodere/decoder to make tests run.

it's in the comment

// TextEncoder and TextDecoder are not available anymore in the jest jsdom environment. They
// are available in all common browsers, therefor we can use them in tests without risk.
// https://github.com/inrupt/solid-client-authn-js/issues/1676

**Review by ewelinajozwik — APPROVED** [review]

**Pull request merged**